composer.json

```diff
@@ -21,6 +21,7 @@
     ],
     "require": {
         "php": "^7.4 || ^8.0",
+        "ext-filter": "*",
```

Comment:
I wonder if anyone will raise complain about it afterwards 😅

Reply:
Isn't it a built-in extension? I've never required it explicitly even though I've been using `filter_var()` many times 🙂.

Reply:
it's enabled by default, but one can have vanilla PHP without ext. I doubt it, thus original message.
one way or another, our SCA was complaining about usage of not declared extension :shrug: